### PR TITLE
Added Spacelift blog

### PR DESCRIPTION
**Pull request by mariuszmichalowski**

### CODE REVIEW

File: data.js [hunk 1]
```diff
@@ -4565,5 +4565,18 @@ module.exports = [
...
+    url: "https://spacelift.io/blog",
+    twitter: "@spaceliftio",
+    tags: [
+      "Terraform",
+      "Ansible",
+      "Kubernetes",
+      "Docker",
+      "DevOps",
+      "AWS"
+    ],
```
Comment (castarco) [inline data.js:4580]: This JSON is broken, there's no closing brace for the object you introduced.

Comment (mariuszmichalowski) [reply data.js:4580]: Sorry, I must have missed it. All fixed now:)

**Comment by markodenic** [comment]
@mariuszmichalowski Very nice looking blog. Well done! Please, put your data somewhere in the middle of the file. Thanks!

**Comment by mariuszmichalowski** [comment]
@markodenic Thanks a lot! I just edited the code and put Spacelift somewhere in the middle of the file, as requested.

**Review by markodenic — APPROVED** [review]

**Pull request merged**